scripts/rollup.config.js

```diff
@@ -0,0 +1,67 @@
+import resolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import babel from 'rollup-plugin-babel';
+import replace from 'rollup-plugin-replace';
+import { uglify } from 'rollup-plugin-uglify';
+import filesize from 'rollup-plugin-filesize';
+
+const version =
+  process.env.VERSION || `UNRELEASED (${new Date().toUTCString()})`;
+const algolia = '© Algolia, Inc. and contributors; MIT License';
+const link = 'https://github.com/algolia/instantsearch.js';
+const license = `/*! InstantSearch.js ${version} | ${algolia} | ${link} */`;
+
+const plugins = [
+  resolve({
+    browser: true,
+    preferBuiltins: false,
+  }),
+  babel({
+    exclude: 'node_modules/**',
+    plugins: [
+      'external-helpers',
+      'transform-react-remove-prop-types',
+      'transform-react-constant-elements',
+      'transform-react-pure-class-to-function',
+    ],
+  }),
+  commonjs(),
+  filesize({
+    showMinifiedSize: false,
+    showGzippedSize: true,
+  }),
+];
+
+const createConfiguration = ({ mode = 'production', minify = true } = {}) => ({
```

Comment:
It seems that the minification depends on the mode so we can drop the minify options.

Reply:
I was waiting for you to agree on the bundle names. If that's OK for you, I can simplify the function.